packages/runner-ct/src/SpecList/SpecGroupItem.tsx

```diff
@@ -0,0 +1,55 @@
+import cs from 'classnames'
+import React, { useState, useEffect } from 'react'
+
+import { CollapseGroup } from './CollapseGroup'
+import { SpecFolder } from '../specs/make-spec-hierarchy'
+import { SpecItem } from './SpecItem'
+import { OnSelectSpec } from './SpecFileItem'
+import './spec-group.scss'
+
+interface SpecGroupProps {
+  group: SpecFolder
+  selectedSpecs: string[]
+  onSelectSpec: OnSelectSpec
+}
+
+export const SpecGroupItem: React.FC<SpecGroupProps> = (props) => {
```

Comment:
It is currently working fine :thinking:. Since everything is passed via props, it seems to work just fine. 

I am actually not clear on what does and does not need to be wrapped in observer. I probably need to spend some more time learning mobx?